hack/clean.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# **clean.sh**
+
+cd $GOPATH/src/github.com/kubeedge/kubeedge
+
+removefolder() {
+if [ -d bin ]; then
+rm -rf bin
+fi
+
+if [ -d .idea ]; then
+rm -rf .idea
+fi
+}
+
+#terminate mosquitto if running
+killmosquitto() {
+if  pgrep mosquitto  >/dev/null 2>&1  ; then
+     pkill -9 mosquitto
+fi
+
+}
+
+#terminate edge core if running
+killedge () {
+if pgrep edge_core >/dev/null 2>&1 ; then
+     pkill -9 edge_core
```

Comment:
How about the edge_core.log file which was generated earlier. After stopping edge_core, move the edge_core.log file to /tmp(possibly) directory and append it with incremental numbers, each time. So that while moving there is not overwriting issue.

Reply:
since we are unaware of the count of the user who does make clean..we cant move it to the other location and incremant the numbers..since the logs get overwritten since we move them.
The alternate way to solve this problem is to start edge core in the following way..
nohup ./edge_core >> edge_core.log 2>&1 & and move the edge_core.log when user does make clean(if its ok with this approach .we can change the command as above in the Readme.md..)

Reply:
Make clean will be executed by a developer when she/he is implementing the code and testing in her/his developmental environment. So don't bother about "since we are unaware of the count of the user who does make clean". Just check edge_core.log.1 exists in /tmp path or not, if so then re-direct the log to edge_core.log.2. 
It is a possibility to append in existing file, but after sometime it will become very huge and not maintainable.

Reply:
@sameersarkar1979 first time it works fine u will check for edge_core.log.1if it present, u will redirect it to  edge_core.log.2.But second time when user does make clean again also it checks edge_core.log1.Since its their it will overwrite to edge_core.log.2, again same problem persists to over come this better to use logrotate as we did in IEF...??
@rohitsardesai83 -Please give inputs on these.

Reply:
The second time edge_core.log.2 should be checked and not edge_core.log.1 again. Then third time edge_core.log.3 should be checked. and so on.

Reply:
log rotation is required , maybe we can introduce this later.

Reply:
@sameersarkar1979 -I have done as you suggested please review

![image](https://user-images.githubusercontent.com/39593583/52463622-97e8ff80-2b9d-11e9-9b03-fc4043e8ad61.png)